### ISSUE TITLE
Disabling the nozzle heating and extruder

### ISSUE THREAD

**Issue by vatecq** [MarlinFirmware/Marlin#18309]
Hello, 

I'm working on a custom designed printer project using the Tevo Tarantula equipments with Easyconfig (1.1.x) firmware. I attached an induction heater on a custom designed reservoir with externel temperature control. I add a composite material to melt inside, manually by hand. 

At this point, I'd like to disable the default **nozzle heating** (and the thermistor) system as well as the **extruder** since I manage these features externally.  How can I disable these from the Configuration file without an error in printing operation. Btw, I'm using Cura slicer. 

Thanks for your kind helps.

**Comment by AnHardt** [comment]
Simplest is - just not using this systems.
Use a fake thermistor 999 or 998.
Edit out all M104/M109 commands out of your g-code-files.

**Comment by vatecq** [comment]
_"> Simplest is - just not using this systems.
> Use a fake thermistor 999 or 998.
> Edit out all M104/M109 commands out of your g-code-files."_
Thanks for your help!
However, how can I apply fake termistor? and should I directly remove these M commands out? Sorry for bothering but I'm a mechanical engineer and cannot manipulate the codes that I need.

**Comment by AnHardt** [comment]
Sorry i will not help any further. The new questions are far below the Niveau i expect here.
Fake thermistors -> Configuration.h. Where else? Configuring is not programming.
G-code-files are human readable and editable with a normal text-editor.

**Comment by vatecq** [comment]
No need to be rude. Me and many people are here for this "simple" questions thanks to the github concept. Besides, thank you so much for helping. I'll try my best.

**Comment by yet-another-average-joe** [comment]
Off topic (not Marlin), but it could help : if you use OctoPrint, the DryRun plugin does exactly what you're asking for, without the need of editing gcode file and Marlin code or config files : https://plugins.octoprint.org/plugins/DryRun/

**Comment by boelle** [comment]
@vatecq still an issue?

**Comment by vatecq** [comment]
@boelle Actually I made a research about applying fake thermistors into config.h and found some good examples. However this is my PhD thesis (turning DIY polymer based printer into full-metal printer) and couldn't risk about editing the firmware because of not having an experience on it. So, I kept the advise of @AnHardt and my findings as a backup plan. 

I solved the issue for now by changing the print temperature to the room temperature and disabled the extruder by opening the gears manually. 

Btw, I was searching for a way totally similar to the comment of @yet-another-average-joe, but for a marlin
